src/components/Lobby/Room/Room.jsx

```diff
@@ -6,26 +6,31 @@ import { startGame, joinRoom, leaveRoom, addAIPlayer, removeAIPlayer } from '../
-        const startGameVisible = !this.props.room.started && inRoom && !(this.props.room.minPlayers - this.props.room.players.length > 0);
-        const waitingVisible = !this.props.room.started && (this.props.room.minPlayers - this.props.room.players.length > 0);
+        const isStarted = this.props.room.started;
+        const hasMaxPlayers = this.props.room.players.length >= this.props.room.maxPlayers;
+        const hasMinPlayers = this.props.room.players.length >= this.props.room.minPlayers;
         return (
-            <div className={styles.room} key={this.props.room.roomId} data-cy="room">
-                <h2 className={styles.roomHeader} data-cy="room-title">{"Room: " + this.props.room.title}</h2>
-                <ul className={styles.roomPlayers} id="players" data-cy='room-players'>
-                    {this.props.room.players.map((player, key) => (
-                        <li className={styles.playerInRoom} key={key}>
-                            <span className={styles.roomPlayer}data-cy='player-username'>{player.username}</span>
-                            {!player.real && <button className={styles.removeAIButton} onClick={() => this.props.removeAIPlayer(this.props.room.roomId, player.username)} data-cy='remove-ai'>-</button>}
-                        </li>
-                    ))}
-                </ul>
-                {waitingVisible && <span className = {styles.waiting} data-cy="players-needed">Waiting for {this.props.room.minPlayers - this.props.room.players.length} more players</span>}
-                <div className={styles.roomButtons}>
-                    {startGameVisible && <button className={styles.roomButton} onClick={this.props.startGame} data-cy="start-game" type='button'>Start game</button>}
-                    {!this.props.room.started && inRoom && <button className={styles.roomButton} onClick={() => this.props.leaveRoom(this.props.room.roomId)} data-cy="leave-room" type='button'>Leave room</button>}
-                    {!this.props.room.started && this.props.room.players.length !== this.props.room.maxPlayers && !inRoom && <button className={styles.roomButton} onClick={() => this.props.joinRoom(this.props.room.roomId)} data-cy="join-room" type='button'>Join room</button>}
-                    {!this.props.room.started && this.props.room.players.length !== this.props.room.maxPlayers && inRoom && <button className={styles.roomButton} data-cy="automated-player" onClick={() => this.props.addAIPlayer(this.props.room.roomId)} >Add AI player</button>}
-                </div>
+            <div className={styles.room} data-cy="room">
+                <h2 className={styles.title} data-cy="room-title">{`Room: ${this.props.room.title}`}</h2>
+                <table className={styles.players}>
+                    <tbody data-cy='room-players'>
+                        {this.props.room.players.map((player, key) => (
+                            <tr className={styles.player} key={key}>
```

Comment:
My only point is that I would avoid using an index as a key but I don't think it's a huge deal.

Reply:
I think we might've had it as player id before but the AI players all have an id of 0 so it wouldn't work